types/mongoose/mongoose-tests.ts

```diff
@@ -1241,15 +1247,15 @@ mongoose.model('').findOne({})
     str.toLowerCase;
   });
 
-mongoose.model('').aggregate()
-  .then(function (arg) {
+mongoose.model('').aggregate([])
+  .then(function (arg: any) {
     return 2;
-  }).then(function (num) {
+  }).then(function (num: any) {
```

Comment:
We should remove the types here (`arg: any`). The return type should be inferred by whatever's returned by the promise.